src/common/helpers/config.js

```diff
@@ -1,40 +1,12 @@
-export class Config {
-  constructor(config) {
-    // https://github.com/purposeindustries/window-or-global/blob/master/lib/index.js
-    // Establish the root object, `window` in the browser, or `global` on the server.
-    let root = typeof self == 'object' && self.Object == Object && self;
-    // root is window, we expect it to have config
-    if (root && !root.__CONFIG__) {
-      throw new ReferenceError('Client config: no config defined.');
-    }
-    // root is not window; test for node global and assign to avoid ref error
-    // we don't expect config here, this is client side only, but we can set it
-    // for unit tests
-    root = root || (typeof global == 'object' && global.Object == Object && global);
+import { Map } from 'immutable';
 
-    this._store = config || root.__CONFIG__;
-  }
+export const conf = Map(getGlobal().__CONFIG__);
```

Comment:
@bartaz: Yes. It makes importing the helper cleaner (because of the shorter path) and it allows us to cleanly separate the implementation. I don't think there's a major reason not to export conf as default, but I guess I prefer helper files with multiple helpers to export them all in the same way ¯\_(ツ)_/¯